src/OnlySpans.PolyLeads.Api/Data/Records/DocumentSeed.cs

```diff
@@ -0,0 +1,3 @@
+﻿namespace OnlySpans.PolyLeads.Api.Data.Records;
+
+public sealed record DocumentSeed(string Name, string Description, Uri DownloadUrl);
```

Comment:
```suggestion
public sealed record DocumentSeed(
    string Name,
    string Description,
    Uri DownloadUrl);
```

у нас такой кодстайл